### ISSUE TITLE
能否将frpc.exe与主程序分离，使Frpc.exe可替换？

### ISSUE THREAD

**Issue by OoyonghongoO** [koho/frpmgr#82]
Sakura panel的强制要求需要用户使用定制的客户端，请尝试做一个能够自行替换frpc.exe的版本，我(们干草Frp)会非常感谢的~

:)

**Comment by Hhpon** [comment]
同求

**Comment by void-gfly** [comment]
#5 同求，frp官方版本0.58-0.6都会不定期断连, 0.57版本稳定使用,frpmgr目前集成的版本是0.6,无法更换没法正常稳定使用

**Comment by renzcu** [comment]
同求

**Comment by wuyilingwei** [comment]
我觉得不如出一个自定义core的功能，每个连接可以自定义版本，如果没有对应版本自动下载

**Comment by MANKUD** [comment]
> [#5](https://github.com/koho/frpmgr/issues/5) Same request, frp official versions 0.58-0.6 will be disconnected from time to time, 0.57 version is stable to use, the current integrated version of frpmgr is 0.6, it cannot be replaced and cannot be used normally and stably

try using these configurations to fix the disconnection issue heartbeat interval set to 30 seconds and the timeout set to 60 seconds

**Comment by MANKUD** [comment]
> [#5](https://github.com/koho/frpmgr/issues/5) 同求，frp官方版本0.58-0.6都会不定期断连, 0.57版本稳定使用,frpmgr目前集成的版本是0.6,无法更换没法正常稳定使用

try using these configurations to fix the disconnection issue heartbeat interval set to 30 seconds and the timeout set to 60 seconds

**Comment by OoyonghongoO** [comment]
> > [#5](https://github.com/koho/frpmgr/issues/5) 同求，frp官方版本0.58-0.6都会不定期断连， 0.57版本稳定使用，frpmgr目前集成的版本是0.6，无法更换没法正常稳定使用
> 
> try using these configurations to fix the disconnection issue heartbeat interval set to 30 seconds and the timeout set to 60 seconds

Thank you for making it more convenient for us to use FRP with your program, but we have now started developing our own launcher. Thank you very much!
感谢您让我们更方便地将FRP与您的程序一起使用，但我们现在已经开始开发我们自己的启动器。非常感谢你！

Regarding disconnection, try setting tls_enable to false to solve the problem.
关于断开连接的问题，可以尝试将tls_enable设置为false以解决该问题。

**Comment by OoyonghongoO** [comment]
> 我觉得不如出一个自定义core的功能，每个连接可以自定义版本，如果没有对应版本自动下载

I agree, this is more convenient and user-friendly, but I'm not sure if the author is willing to create such a feature.
赞同，这样更加方便也更加好用，但是不知道作者是否愿意做一个这样的功能。
